### PR TITLE
Improved README.md

### PR DESCRIPTION
**Pull request by DominikaK**
Updated links and corrected typos.

### CODE REVIEW

**Comment by andrerom** [comment]
Looks ok but as ezstudo is a fork of ezplatform, so we should first apply fixes there and merge up to:
- ezplatform-demo
- ezstudio
 - ezstudio-demo *(which is a fork of ezstudio)* 

And there are some changes last week by david which was just merged that might conflict with this: https://github.com/ezsystems/ezplatform/commit/c1e5ae106ed230a66336fb0d6f7f53a0742330f1

**Comment by DominikaK** [comment]
Readmes are distinct pages for Platform and Studio, and David's changes are not in sections that are common for the two, so there should be no conflict.

**Comment by andrerom** [comment]
@DominikaK merged the changes in, seems like there is a conflict ;)


@lserwatka note that I did not merge into studio demo, as I assume this will be fixed, merged and merged up there with the other doc changes.

**Comment by DominikaK** [comment]
Fixed (hopefully, thanks to Rafał's help).

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # eZ Studio
 
 ## What is eZ Studio ?
-*eZ Studio* is a commercial CMS (Content Management System) software developed by eZ Systems.
+*eZ Studio* is commercial CMS (Content Management System) software developed by eZ Systems AS.
```
Comment (andrerom) [inline README.md]: "AS" should be omitted, it was removed by change from @DavidLiedle

File: README.md [hunk 2]
```diff
@@ -40,11 +40,11 @@ In short eZ Studio comes with new features and services that extend eZ Platform
...
 
 ### Further information:
-*eZ Studio* is a commercial software which adds advanced features for editorial teams and media companies, 100% built on top of *eZ Platform* APIs.
+*eZ Studio* is commercial software which adds advanced features for editorial teams and media companies, 100% built on top of *eZ Platform* APIs.
 
-- eZ Platform Developer Hub: [ezplatform.com](https://ezplatform.com/)
-- [eZ Studio and eZ Platform RoadMap](http://doc.ez.no/roadmap)
-- eZ Systems (commercial products and services): [ez.no](http://ez.no/)
+- [eZ Studio Architecture, including "Platform Stack"](https://doc.ez.no/display/DEVELOPER/Architecture:+An+Open+Source+PHP+CMS+Built+On+Symfony2+Full+Stack)
+- eZ Systems AS *(commercial products and services)*: [ez.no](http://ez.no)
```
Comment (andrerom) [inline README.md]: drop As

File: README.md [hunk 3]
```diff
@@ -40,11 +40,11 @@ In short eZ Studio comes with new features and services that extend eZ Platform
 - **PlatformUIBundle**: A modern, extensible "backend" UI for managing content & administering the site
 
 ### Further information:
-*eZ Studio* is a commercial software which adds advanced features for editorial teams and media companies, 100% built on top of *eZ Platform* APIs.
+*eZ Studio* is commercial software which adds advanced features for editorial teams and media companies, 100% built on top of *eZ Platform* APIs.
 
-- eZ Platform Developer Hub: [ezplatform.com](https://ezplatform.com/)
-- [eZ Studio and eZ Platform RoadMap](http://doc.ez.no/roadmap)
```
Comment (andrerom) [inline README.md]: theses two links added by @DavidLiedle  might make sense here to

**Comment by lserwatka** [comment]
@andrerom I will fixed this merge up to ezstudio-demo

**Pull request merged**